client/state/countries/reducer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { SMS, DOMAIN, PAYMENT } from './constants';
+import {
+	COUNTRIES_RECEIVE,
+	COUNTRIES_REQUEST,
+	COUNTRIES_REQUEST_FAILURE,
+	COUNTRIES_REQUEST_SUCCESS
+} from 'state/action-types';
+import { countriesSchema } from './schema';
+import { createReducer } from 'state/utils';
+
+export const items = createReducer( { [ SMS ]: [], [ DOMAIN ]: [], [ PAYMENT ]: [] }, {
```

Comment:
> We could use the same event internally, but I haven't seen encountered something like that either.

We have a few instances, but usually they usually update different parts of the tree. For example `PLANS_RECEIVE` updates both the current-user currencyCode reducer and in the plans reducer. Another example would be `SITE_SETTINGS_RECEIVE` which is used in a number of places.